docs/troubleshoot/verify-fingerprint.md

```diff
@@ -147,50 +154,56 @@ Info: Result files and script log are in directory /u/username/hash-v1.12.0
+```
+zowe-verify-authenticity.sh [-r <runtime-dir>] [-h <HashPgm-dir>] [-f <HashRef-dir>] [-l <output-dir>]
+```
+- All parameters are optional
+- You can use dot (.) and tilde (~) in the parameters
 
-Description of parameters
+Description of parameters:
 
-   - All parameters are optional
+- `-r <runtime-dir>` 
+   
+   Root directory of the executables used by Zowe at run time. The typical value is `/usr/lpp/zowe`. The default value is the parent directory of the `bin` folder where this script is located.
 
-   - You can use dot (.) and tilde (~) in the parameters
+- `-h <HashPgm-dir>` 
 
-`-r <runtime-dir>` root directory of the executables used by Zowe at run time
+   Directory of the hash key program. The typical value is `/usr/lpp/zowe/fingerprint`. The default value is the `fingerprint` directory of the parent folder where this script is located.
 
-typical value : `/usr/lpp/zowe`
-default: the parent directory of the 'bin' folder where this script is located
+- `-f <HashRef-dir>` 
 
-`-h <HashPgm-dir>` directory of the hash key program
+   Directory of the reference hash key file `RefRuntimeHash-V.v.p.txt`. The typical value and default value are the same as that of the `-h` parameter. The values specified for `-h` and `-f` can be the same or different.
 
-typical value : `/usr/lpp/zowe/fingerprint`
-default: the 'fingerprint' directory of the parent folder where this script is located
+- `-l <output-dir>`  
 
-`-f <HashRef-dir>` directory of the reference hash key file `RefRuntimeHash-v.r.m.txt`
-same typical value and default as -h
-The values specified for `-h` and `-f` can be the same or different.
+   Output directory where the following log and output files will be written.
+   ```
+   zowe-verify-authenticity.log
+   CustRuntimeHash.sort
+   CustRuntimeHash.txt 
+   RefRuntimeHash.sort  
+   ```
+   
+   The typical value is `~/zowe/fingerprint`. The directory will be created if you specify it but it does not exist.
 
-`-l <output-dir>`  output directory where the following log and output files will be written.
-```
-    zowe-verify-authenticity.log
-    CustRuntimeHash.sort
-    CustRuntimeHash.txt 
-    RefRuntimeHash.sort  
-```
-typical value : `~/zowe/fingerprint`
-The directory will be created if you specify it but it does not exist.
+   The following defaults will be tried in the listed order: 
+   ```
+   /global/zowe/log 
+   ```
+   ```
+   ~/zowe
+   ``` 
+   ```
+   $TMPDIR
+   ```
+   ```
+   /tmp
+   ```
 
-default: The following defaults will be tried in this order
-```
-    /global/zowe/log 
-    ~/zowe 
-    $TMPDIR
-    /tmp
-```
 ## Use of zowe-verify-authenticity.sh by zowe-support.sh
 
-From v1.14 on, `zowe-verify-authenticity.sh` is automatically called, with no parameters, by `zowe-support.sh`.  
+Starting in Zowe v1.14, the `zowe-verify-authenticity.sh` script is automatically called, with no parameters, by `zowe-support.sh`.  
```

Comment:
This is what I intend, but it may not make it into the 1.14 delivery, although this is unlikely.

Reply:
Let's revisit this text before 1.14 is published. : )